README.md

```
@@ -80,20 +65,26 @@ Thank you for contributing to ByteBeasts Frontend! If you have any questions, fe
-      <img src="Maintainers/JuanDi.jpeg" width="100px;" alt="Maintainer: JuanDi"/>
+      <img src="Maintainers/juandiEverai.jpeg" width="100px;" alt="Maintainer: JuanDi"/>
       <br />
       <a href="https://t.me/JuanDixCode">JuanDi</a>
       <br />
     </td>
   </tr>
 </table>
 
-<img width="928" alt="Captura de pantalla 2024-08-05 a la(s) 6 20 29 p  m" src="https://github.com/user-attachments/assets/672227b4-4081-44cd-9e6b-d36d71925058">
-<img width="928" alt="Captura de pantalla 2024-08-05 a la(s) 6 20 54 p  m" src="https://github.com/user-attachments/assets/51c08c44-7284-403d-888a-2fda4957d996">
+<img width="928" src="Gameplay/forest1.png">
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)